.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+    - package-ecosystem: npm
+      directory: '/'
+      schedule:
+          interval: 'weekly'
+      open-pull-requests-limit: 10
+      commit-message:
```

Comment:
I think we can remove the `commit-message` config here. I'm sure the defaults are fine.